vendor/lowrisc_ip/ip/prim/prim.core

```diff
@@ -21,7 +21,7 @@ filesets:
       - lowrisc:prim:flop_2sync
       - lowrisc:prim:arbiter
       - lowrisc:prim:fifo
-      - lowrisc:prim:alert
+#      - lowrisc:prim:alert
```

Comment:
Is this intentional? If so it will need to go in via a vendoring patch

Reply:
This was because alert primitive is dependent on `lc_ctrl_pkg` (from OpenTitan). Problem comes up specifically with this PR because `ram_1p_scr` module is dependent on `prim:all` and up to this point not many `.core` files were dependent on `prim:all` in Ibex.

Reply:
https://github.com/lowRISC/opentitan/pull/9306 and https://github.com/lowRISC/opentitan/pull/9304 does the trick for this PR. Thanks @rswarbrick for the help!